### ISSUE TITLE
How to provide large files ?

### ISSUE THREAD

**Issue by Thibault-Poinsignon** [data-fun/3d-scere#1]
For now, `static/adjacency_matrix_V4.parquet.gzip` (412,3 MB) and `static/SCERE.db` (354,7 MB) are in `.gitignore`. How should we proceed to make them available through the GitHub repository ?

**Comment by pierrepo** [comment]
They probably won't be in a GitHub repo but deposited in Zenodo (later).

**Comment by Thibault-Poinsignon** [comment]
With a link in the readme then ?

**Comment by pierrepo** [comment]
Exactly, in the README, we will have a section related to data preparation / required data in which we will explain how to get these data.
This will probably be a command like this:
```bash
wget https://zenodo.org/blablablab/something.gzip
```
